framework/console/controllers/HelpController.php

```
@@ -52,7 +53,21 @@ public function actionIndex($command = null)
             $result = Yii::$app->createController($command);
             if ($result === false) {
                 $name = $this->ansiFormat($command, Console::FG_YELLOW);
-                throw new Exception("No help for unknown command \"$name\".");
+                $message = "No help for unknown command \"$name\".";
+                try {
+                    $commands = RouteHelper::getRoutes(true);
+                    if ($alternatives = RouteHelper::find($command, $commands)) {
+                        if (count($alternatives) == 1) {
+                            $message .= sprintf("\nDid you mean \"%s\"?", reset($alternatives));
+                        } else {
+                            $message .= "\nDid you mean one of these?\n    " . implode("\n    ", $alternatives);
+                        }
+                    }
+                } catch (\Exception $exc) {
+
```

Same here. Looks suspicious. Do we need to ignore exceptions here?